docs/tutorial/mac-app-store-submission-guide.md

```diff
@@ -1,29 +1,28 @@
 # Mac App Store Submission Guide
 
-Since v0.34.0, Electron allows submitting packaged apps to Mac App Store (MAS),
-this guide provides information on how to submit your app, and the limitations
+Since v0.34.0, Electron allows submitting packaged apps to the Mac App Store (MAS).
+This guide provides information on: how to submit your app & the limitations
```

Comment:
> &

Let's go with "and" here because I think that's more common in the rest of the docs.